test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,17 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+    
+    public static void compareFiles(Path path1, Path path2) throws IOException {
+        Scanner scan1 = new Scanner(path1);
+        Scanner scan2 = new Scanner(path2);
```

Comment:
Seems like [`Files.readAllLines()`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-java.nio.charset.Charset-) will be much more succinct.

Take note of the decoding charset though. I think it needs to be `Charset.defaultCharset()` rather than UTF-8.